.github/workflows/test.yml

```
@@ -96,23 +90,18 @@ jobs:
             os: windows-latest
             tox_env: "py312"
 
-          - name: "ubuntu-py37"
-            python: "3.7"
-            os: ubuntu-latest
-            tox_env: "py37-lsof-numpy-pexpect"
-            use_coverage: true
-          - name: "ubuntu-py37-pluggy"
-            python: "3.7"
-            os: ubuntu-latest
-            tox_env: "py37-pluggymain-pylib-xdist"
-          - name: "ubuntu-py37-freeze"
-            python: "3.7"
-            os: ubuntu-latest
-            tox_env: "py37-freeze"
           - name: "ubuntu-py38"
             python: "3.8"
             os: ubuntu-latest
             tox_env: "py38-xdist"
```

Seems we are no longer running this variant that we were running with py37:

```suggestion
            tox_env: "py38-lsof-numpy-pexpect"
            use_coverage: true
```